OrbitGl/TimeGraph.cpp

```diff
@@ -1068,6 +1068,15 @@ void TimeGraph::JumpToNeighborBox(const TextBox* from, JumpDirection jump_direct
   }
 }
 
+void TimeGraph::SetRightMargin(float margin) {
+  {
+    if (right_margin_ != margin) {
+      NeedsUpdate();
+    }
+    right_margin_ = margin;
```

Comment:
I would like to have this line inside the if-condition

Reply:
I'd be fine with both cases, but I really think the name should be `UpdateRightMargin`.